server/src/main/java/org/apache/iotdb/db/metadata/schemaregion/SchemaEngine.java

```diff
@@ -60,6 +64,10 @@ public SchemaRegion getSchemaRegion(ConsensusGroupId schemaRegionId) {
     return schemaRegionMap.get(schemaRegionId);
   }
 
+  public SchemaRegionStateMachine getOrCreateDataRegionStateMachine(ConsensusGroupId gid) {
```

Comment:
should be getOrCreateSchemaRegionStatemachine.

Reply:
Fixed